errata/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+  <!--
+    The data-githubrepo attribute provides the owner/repo name on github. For W3C repositories most of those are of the
+    form 'w3c/XXX', although there are groups that have their own owner for their repository.
+  -->
+  <head data-githubrepo="w3c/dxwg">
+    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+    <title>Open Errata for the Dataset Exchange Working Group</title>
+    <link rel="stylesheet" type="text/css" href="https://w3c.github.io/display_errata/assets/errata.css"/>
+    <script src="https://w3c.github.io/display_errata/assets/errata.js" type="text/javascript"></script>
+    <script src="https://w3c.github.io/display_errata/assets/toc.js" type="text/javascript"></script>    
+
+    <style type="text/css">
+      .todo {
+        background-color: yellow
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <p class="banner"><a accesskey="W" href="/"><img width="72" height="48" alt="W3C" src="https://www.w3.org/Icons/w3c_home" /></a> </p>
+      <br />
+      <h1 class="title" class="done">Open Errata for the Dataset Exchange Working Group</h1>
+      <dl>
+        <dt>Latest errata update:</dt>
+        <dd><span id="date"></span></dd>
+        <dt>Number of recorded errata:</dt>
+        <dd><span id="number"></span></dd>
+        <dt>Link to all errata:</dt>
+        <dd><span id="errata_link"></span></dd>
+      </dl>
+
+      <section data-notoc>
+        <h1>How to Submit an Erratum?</h1>
+        <p>Errata are introduced and stored in the <a class="done" href="https://github.com/w3c/dxwg/issues/">issue list of the group‘s GitHub repository</a>. The workflow to add a new erratum is as follows:</p>
+        <ul>
+           <li>An issue is raised for a possible erratum. The label of the issue SHOULD be set to “<code>ErratumRaised</code>”. It SHOULD also include the label corresponding to the document on which the erratum is raised, e.g., “<code class="done">dcat</code>” or “<code class="todo">@@@@</code>”. (Please, consult the <a class="done" href="https://github.com/w3c/dxwg/labels">list of available labels</a>.) It is o.k. for an erratum to have several labels. In some, exceptional, cases, i.e., when the erratum is very general, it is also acceptable not to have a reference to a document.</li>
```

Comment:
At the moment, we need it for DCAT only. We can update with other recs as soon as they are published.
  
```suggestion
           <li>An issue is raised for a possible erratum. The label of the issue SHOULD be set to “<code>ErratumRaised</code>”. It SHOULD also include the label corresponding to the document on which the erratum is raised, i.e,, “<code class="done">dcat</code>”. (Please, consult the <a class="done" href="https://github.com/w3c/dxwg/labels">list of available labels</a>.) It is o.k. for an erratum to have several labels. In some, exceptional, cases, i.e., when the erratum is very general, it is also acceptable not to have a reference to a document.</li>
```